app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,13 @@
+:ruby
+  custom_services = @client.custom_services.
+    joins(enrollment: :client).
+    joins(custom_service_type: :custom_service_category).
+    merge(GrdaWarehouse::Hud::Enrollment.visible_to(current_user)).
+    preload( :enrollment, :project, :client, :custom_service_type ).
+    preload(custom_service_type: :custom_service_category).
+    order(date_provided: :desc)
+- if custom_services.any?
+  = render 'hmis_client/services/table', custom_services: custom_services
```

Comment:
I don't see much benefit of rendering another partial here as opposed to just writing the table view directly in this file. It seems like we do that for many other rollups. But that might just be me, it is always hard for me to find referenced views in my IDE.

Reply:
Oh I see, its because its in the driver. Even harder to find 😬  but I appreciate sticking to a pattern, so its OK with me.